src/features/contactmoment/ContactmomentVragenMenu.vue

```diff
@@ -70,31 +72,27 @@ async function waitForConfirmation() {
     display: flex;
     align-items: stretch;
     aspect-ratio: 1;
-
-    &:last-of-type {
-      grid-column: -2 / -1;
-    }
+    font-weight: bold;
   }
 
   button,
   span {
     display: flex;
     background: none;
-    margin: 0;
     padding: 0;
+    margin: 0;
     border: none;
     align-items: center;
     justify-content: center;
     width: 100%;
-  }
-
-  button {
-    color: var(--color-white);
+    font: inherit;
+    border-radius: 100%;
+    height: unset;
+    color: inherit;
   }
 
   span {
     background: var(--color-white);
```

Comment:
This actually is the active state of one of the buttons, however, within this css it's quite hard to understand that without looking at the above template.

I did some testing, if we implement the changes as mentioned in the template, the following css would render the same result and adds a disabled state of the current vraag button:

```
.vragen-menu {
  display: flex;
  flex-wrap: wrap;
  gap: var(--spacing-default);
  padding: var(--spacing-small);

  button {
    all: unset;
    display: flex;
    align-items: center;
    justify-content: center;
    width: var(--spacing-large);
    height: var(--spacing-large);
    border-radius: 50%;
    font-weight: bold;

    &.current {
      background: var(--color-white);
    }

    &:hover {
      cursor: pointer;
    }

    &:disabled {
      cursor: not-allowed;
    }
  }
}
```

Reply:
This way we don't have to style the `li` element with `aspect-ratio` either.

Reply:
Fair point, done!